src/libstd/collections/mod.rs

```diff
@@ -412,18 +412,25 @@
 #[rustc_deprecated(reason = "moved to `std::ops::Bound`", since = "1.26.0")]
 #[doc(hidden)]
 pub use crate::ops::Bound;
+
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use alloc_crate::collections::{binary_heap, btree_map, btree_set};
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use alloc_crate::collections::{linked_list, vec_deque};
+
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(inline)]
 pub use alloc_crate::collections::{BTreeMap, BTreeSet, BinaryHeap};
+
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(inline)]
 pub use alloc_crate::collections::{LinkedList, VecDeque};
 
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(inline)]
 pub use self::hash_map::HashMap;
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(inline)]
 pub use self::hash_set::HashSet;
```

Comment:
Is there any doc attribute I could use to hide `hash_map::HashMap` from search
but leave item at `hash_map::HashMap` clickable?

Reply:
It would require creating a new one. Do you mind creating an issue and tag the rustdoc team on it so we can discuss it please?

Reply:
I opened #67368 for discussion.